doc/source/param/method.incl

```diff
@@ -74,6 +75,64 @@ flux_correct
 
 :e:`Flux correction must be applied to conserved fields in AMR simulations to maintain conserved quantities across mesh resolution jumps.  This parameter selects the group of fields to which the "flux_correct" method will be applied.`
 
+:e:`Fields that store a conserved quantity divided by density`
+(e.g. ``"total_energy"``, ``"velocity_x"``) :e:`have special
+handling. Such fields must be included in both the group specified by
+this parameter AND the` ``"make_field_conservative"`` :e:`group. Flux
+corrections are applied to an element of such fields according to the
+following procedure:`
+
+  1. :e:`The element is multiplied by the corresponding element of
+     the` ``"density"`` :e:`field (before the flux corrections are
+     applied to the` ``"density"`` :e:`field).`
+
+  2. :e:`Flux corrections are applied to the product from step 1.`
+
+  3. :e:`Finally, the element in the original field is assigned the
+     value computed in step 2 divided by the corresponding element
+     from the` ``"density"`` :e:`field (after flux corrections are
+     applied to the` ``"density"`` :e:`field).`
+
+:e:`An error will be raised if these special fields are detected, and
+the` ``"density"`` :e:`field is not included in the group specified by
+this parameter.`
+
+----
+
+:Parameter:  :p:`Method` : :p:`flux_correct` : :p:`min_digits`
+:Summary: :s:`Number of digits to be conserved by fields in tests`
+:Type:    :t:`list`
+:Default: :d:`[]`
+:Scope:     :z:`Cello`
+
+:e:`Specifies the minimum number of digits that are expected to be conserved
+by fields. This is used for testing purposes (the simulation will
+check at each timestep whether this expectation has been met). Entries of this list should alternate between the name of fields (a string) and the expected number of conserved digits for that field (a float).`
```

Comment:
Currently, yes it's always performed (that behavior predates my changes).

We can definitely make this definitely change, but that would involve either changing the existing behavior of some parameters or introducing a new parameter. I would rather not do that in this PR because I'm a little concerned that we might break some of James's tests.